lib/css/components/_stacks-navigation.less

```diff
@@ -8,9 +8,46 @@
 //
 //  TABLE OF CONTENTS
 //  • BASE STYLE
+//  • MUTED STYLE
+
+
+& { // create a single scope to load the config into, at least for now
+    #stacks-internals #load-config();
+
+//      MIXIN:
+//      Navigation item
 //
+//      HOW IT WORKS:
+//      Defines the s-navigation--item with the colors to be used. 
+//      The default values are the ones used by the base style.
+//      Style modifications can pass different colors as needed.
+//  ----------------------------------------------------------------------------
+.s-navigation--item(@fc: @black-600, @hover-bg: @black-100, @hover-fc: @black-600, @selected-bg: @navigation-link-highlight, @selected-fc: @white){
```

Comment:
I don't think we need to be this clever. We can simply add the overrides with

```
&.s-navigation__muted .s-navigation--item {
    &:hover {
        // style
    }
}
```

This way, you aren't duplicating the entirety of the component's CSS, only the overrides... Unless I'm not understanding how Less compiles in this context. Every byte counts.